osu.Game/Screens/Edit/Components/RadioButtons/RadioButton.cs

```diff
@@ -4,16 +4,27 @@
 using System;
 using osu.Framework.Bindables;
 using osu.Framework.Graphics;
+using osu.Framework.Localisation;
 
 namespace osu.Game.Screens.Edit.Components.RadioButtons
 {
     public class RadioButton
     {
         /// <summary>
         /// Whether this <see cref="RadioButton"/> is selected.
+        /// Disable this bindable to disable the button.
         /// </summary>
         public readonly BindableBool Selected;
 
+        /// <summary>
+        /// Tooltip text that will be shown on hover if button is enabled.
+        /// </summary>
+        public LocalisableString TooltipTextWhenEnabled { get; set; } = string.Empty;
+        /// <summary>
+        /// Tooltip text that will be shown on hover if button is disabled.
+        /// </summary>
+        public LocalisableString TooltipTextWhenDisabled { get; set; } = string.Empty;
```

Comment:
these can't exist in their current form, especially so that they're declared on `RadioButton` _but only a subclass of `RadioButton` uses these_. but i'd argue that they should not exist at all

Reply:
Fixed, simplified them to one single TooltipText property. Showing only on disabled is up to the consumer now (through Enabled bindable)